.github/workflows/Manager-CI.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:    
       - 'main'
     paths:
+      - '.github/workflows/Manager-CI.yaml'
```

Comment:
Can you explain why we need this path?

Reply:
Manager-CI check should be launched also when `.github/workflows/Manager-CI.yaml'` is modified.